osu.Game/osu.Game.csproj

```diff
@@ -50,4 +49,7 @@
     <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />
     <PackageReference Include="System.Runtime.Handles" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\osu-resources\osu.Game.Resources\osu.Game.Resources.csproj" />
```

Comment:
Probably unintentional? (in other places too)

Reply:
yeah, need to drop that commit.